tldw_Server_API/app/core/AuthNZ/repos/managed_secret_refs_repo.py

```
@@ -0,0 +1,387 @@
+        if isinstance(value, dict):
+            return dict(value)
+        if isinstance(value, str) and value.strip():
+            try:
+                parsed = json.loads(value)
+            except (TypeError, ValueError):
+                return {}
+            return dict(parsed) if isinstance(parsed, dict) else {}
+        return {}
+
+    @classmethod
+    def _row_to_dict(cls, row: Any) -> dict[str, Any]:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `row`

(ANN401)